server/src/app.routes.ts

```diff
@@ -40,17 +46,17 @@ router.post("/base/save", auth, debugDataLog("Base save data"), baseSave);
 router.post("/base/updatesaved", auth, debugDataLog("Base updated save"), updateSaved);
 
 // Yard Planner
-router.get("/api/bm/yardplanner/gettemplates", auth, debugDataLog("Get templates"), getTemplates);
-router.post("/api/bm/yardplanner/savetemplate", auth, debugDataLog("Saving template"), saveTemplate);
+router.get("/api/:apiVersion/bm/yardplanner/gettemplates", apiVersion, auth, debugDataLog("Get templates"), getTemplates);
+router.post("/api/:apiVersion/bm/yardplanner/savetemplate", apiVersion, auth, debugDataLog("Saving template"), saveTemplate);
 
 // Inferno
-router.post("/api/bm/base/load", auth, debugDataLog("Inferno load data"), baseLoad);
-router.post("/api/bm/base/infernomonsters", auth, debugDataLog("Load inferno monsters"), infernoMonsters);
-router.post("/api/bm/base/save", auth, debugDataLog("Inferno save data"), baseSave);
+router.post("/api/:apiVersion/bm/base/load", apiVersion, auth, debugDataLog("Inferno load data"), baseLoad);
+router.post("/api/:apiVersion/bm/base/infernomonsters", apiVersion, auth, debugDataLog("Load inferno monsters"), infernoMonsters);
+router.post("/api/:apiVersion/bm/base/save", apiVersion, auth, debugDataLog("Inferno save data"), baseSave);
 
 // Worldmap v2
 router.post("/worldmapv2/getarea", auth, debugDataLog("MR2 get area"), getArea);
-router.post("/api/player/savebookmarks", auth, debugDataLog("MR2 save bookmarks"), getArea);
+router.post("/api/:apiVersion/player/savebookmarks", auth, debugDataLog("MR2 save bookmarks"), getArea);
```

Comment:
MIssing apiVersion middleware

Reply:
Fixed in [latest commit](https://github.com/bym-refitted/backyard-monsters-refitted/commit/4d306e150f1f7dfb4c92ef94d6f2c0084345cb08)